openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
+oeps: {}
+openedx-release: {parent-repo: edx/edx-analytics-dashboard}
+owner: edx/analytics
+tags: [analytics]
```

Comment:
This seems to be a different format than what's specified in [oep-2](https://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html).  It seems like it should be something more like:

```
tags:
    - analytics
```

Is that right?